TemplateFiles/DeploymentFiles/Add-AztsFeatureConfigurationValues.ps1

```diff
@@ -0,0 +1,254 @@
+﻿<###
```

Comment:
After the feedback for polishing, can you send the new screenshots for these scripts?

Reply:
Please find the screenshot for output window along with log file below:
 
For Enabling Feature: [AzTS Enable/Disable Feature Script Demo](onenote:https://microsoft.sharepoint.com/teams/ISRMSAFETIndia/Shared%20Documents/AzSK-AzureTenantSecurity/AztsEnableDisableFeature.one#AzTS%20Enable/Disable%20Feature%20Script%20Demo&section-id=%7B55F46BAE-74A9-4A88-A6A9-3C9B5D8B00FD%7D&page-id=%7B80D1AA2F-0DF4-48D5-A0C8-D496E9FD1F0D%7D&end)  ([Web view](https://microsoft.sharepoint.com/teams/ISRMSAFETIndia/_layouts/OneNote.aspx?id=%2Fteams%2FISRMSAFETIndia%2FShared%20Documents%2FAzSK-AzureTenantSecurity&wd=target%28AztsEnableDisableFeature.one%7C55F46BAE-74A9-4A88-A6A9-3C9B5D8B00FD%2FAzTS%20Enable%5C%2FDisable%20Feature%20Script%20Demo%7C80D1AA2F-0DF4-48D5-A0C8-D496E9FD1F0D%2F%29&xsdata=MDV8MDF8fGQ4NDcwOWQ1MTJiMDRhYmE3ZWIzMDhkYmRiYThlZjc0fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgzNDUyOTA4MjE0MzUwNTl8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPalJqTkdGa01URXlMVFZoTXpFdE5ERTVOeTA0TW1NMUxUZGtaRFF5WkRrM01qaGpNMTltTXpZd1pqaG1OaTA1TlRobExUUTRNR0l0T0RkbU55MW1OVGt3T1RnME5ETXlNVGxBZFc1eExtZGliQzV6Y0dGalpYTXZiV1Z6YzJGblpYTXZNVFk1T0Rrek1qSTRNVGswTUE9PXwyYTBmMzc5MjEwYTI0Njg1N2ViMzA4ZGJkYmE4ZWY3NHw1MmI4M2I2NmNjYjE0YjQxYWJjNWI3NDgxYTliZDI1Yw%3D%3D&sdata=ODByTHRxUWpveWxlQkxxSXpuNWpmQ3drMkpYOWREMmx5TDlTbG80dkF4MD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-rahkuma%40microsoft.com))
 
For adding Configuration values: [AzTS addition of Feature Configuration Values](onenote:https://microsoft.sharepoint.com/teams/ISRMSAFETIndia/Shared%20Documents/AzSK-AzureTenantSecurity/AztsEnableDisableFeature.one#AzTS%20%20addition%20of%20Feature%20Configuration%20Values&section-id=%7B55F46BAE-74A9-4A88-A6A9-3C9B5D8B00FD%7D&page-id=%7B009E3718-4657-4D87-93F5-2096ACC0D521%7D&end)  ([Web view](https://microsoft.sharepoint.com/teams/ISRMSAFETIndia/_layouts/OneNote.aspx?id=%2Fteams%2FISRMSAFETIndia%2FShared%20Documents%2FAzSK-AzureTenantSecurity&wd=target%28AztsEnableDisableFeature.one%7C55F46BAE-74A9-4A88-A6A9-3C9B5D8B00FD%2FAzTS%20%20addition%20of%20Feature%20Configuration%20Values%7C009E3718-4657-4D87-93F5-2096ACC0D521%2F%29&xsdata=MDV8MDF8fGQ4NDcwOWQ1MTJiMDRhYmE3ZWIzMDhkYmRiYThlZjc0fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgzNDUyOTA4MjE0MzUwNTl8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPalJqTkdGa01URXlMVFZoTXpFdE5ERTVOeTA0TW1NMUxUZGtaRFF5WkRrM01qaGpNMTltTXpZd1pqaG1OaTA1TlRobExUUTRNR0l0T0RkbU55MW1OVGt3T1RnME5ETXlNVGxBZFc1eExtZGliQzV6Y0dGalpYTXZiV1Z6YzJGblpYTXZNVFk1T0Rrek1qSTRNVGswTUE9PXwyYTBmMzc5MjEwYTI0Njg1N2ViMzA4ZGJkYmE4ZWY3NHw1MmI4M2I2NmNjYjE0YjQxYWJjNWI3NDgxYTliZDI1Yw%3D%3D&sdata=ODByTHRxUWpveWxlQkxxSXpuNWpmQ3drMkpYOWREMmx5TDlTbG80dkF4MD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-rahkuma%40microsoft.com))